packages/picasso.js/src/core/component/brushing.js

```diff
@@ -50,6 +50,7 @@ export function styler(obj, { context, data, style, filter, mode }) {
     const len = nodes.length;
     let nodeData;
     let globalChanged = false;
+    const evaluatedDataProps = typeof dataProps === 'function' ? dataProps(brusher) : dataProps;
```

Comment:
I'm guessing that you need the brush instance to solve your particular case, but to just send that is a gonna get us in to trouble later if we suddenly need to solve some other case and need some other data. So I suggest that you at least pass in an object here instead with a `brush` property that contains the `brusher`.

Reply:
brusher is an object which has a lot of properties and methods like brushes(), containsMappedData(), isActive(). I don't know if we change from brusher to {brush: brusher}, the naming is better?

Reply:
It's a matter of consistency. So `brush` is better than `brusher` IMO as we use `brush` else to describe the brush context. The other alternative I would think is ´{ brushContext: brusher }´

Reply:
Updated!